index.html

```diff
@@ -308,5 +313,7 @@ <h1 class="white-header">FOSSASIA on social media</h1>
     <script src="//use.fontawesome.com/7f8b61349a.js"></script>
     <!-- Github Cards -->
     <script src="//unpkg.com/github-card@1.2.1/dist/widget.js"></script>
+	<!--Flickr Album-->
+	<script src="//flickrembed.com/embed_v2.js.php?source=flickr&layout=responsive&input=www.flickr.com/photos/fossasia&sort=4&by=user&theme=carousel&scale=fit&skin=default&id=584390e95ef74"></script>
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/index.html
+++ b/index.html
@@ -314,6 +314,6 @@
     <!-- Github Cards -->
     <script src="//unpkg.com/github-card@1.2.1/dist/widget.js"></script>
 	<!--Flickr Album-->
-	<script src="//flickrembed.com/embed_v2.js.php?source=flickr&layout=responsive&input=www.flickr.com/photos/fossasia&sort=4&by=user&theme=carousel&scale=fit&skin=default&id=584390e95ef74"></script>
+    <script src="//flickrembed.com/embed_v2.js.php?source=flickr&layout=responsive&input=www.flickr.com/photos/fossasia&sort=4&by=user&theme=carousel&scale=fit&skin=default&id=584390e95ef74"></script>
 </body>
 </html>
```